qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -939,24 +942,41 @@ def _build_qeom_result(
             groundstate_result.eigenvalues[0], excited_eigenenergies
         )
 
-        qeom_result.eigenstates = np.array([])
-
         eigenstate_result = EigenstateResult.from_result(qeom_result)
         result = problem.interpret(eigenstate_result)
 
         return result
 
 
 class QEOMResult(EigensolverResult):
-    """The results class for the QEOM algorithm."""
+    """The results class for the qEOM algorithm.
+
+    For more details about the definitions, please refer to https://arxiv.org/abs/1910.12890.
+
+    Attributes:
```

Comment:
Can we add types to these fields - it goes in brackets after the name before the : (examples can be found elsewhere in nature). I think I would avoid putting None here - I don't think its really a valid value as such (unless it is in which case we should have it there) more that this is just as aspect of how the class is initialized one field at a time.